docs/source/user_guide/tutorials.rst

```diff
@@ -3,21 +3,69 @@
 Tutorials
 =========
 
-The Landlab Tutorials are Jupyter notebooks that give examples of Landlab core
-concepts, component introductions, and teaching exercises.
+The Landlab Tutorials provide examples of Landlab core concepts and component
+introductions. Tutorials exist as interactive Jupyter notebooks that contain
+alternating cells of computer code and text that explain the code. In addition
+to Landlab Tutorials that exemplify Landlab, notebooks intended to teach and
+learn surface dynamics are the
+:ref:`Landlab Teaching Tutorials <teaching_tutorials>`.
 
-You can run the Tutorials using your local Landlab installation. The Tutorials
-are located within the main Landlab repository in the
-`notebooks <https://github.com/landlab/landlab/tree/master/notebooks>`_ folder.
+Launch notebooks online
+-----------------------
 
-Landlab Tutorials can also be accessed online with the source code stored on
-Github. Launch these notebooks with the following link:
+Landlab Notebooks can be accessed online with the following link:
 `Binder <https://mybinder.org/v2/gh/landlab/landlab/release?filepath=notebooks/welcome.ipynb>`_.
+Here the notebooks are provided within a binder environment that includes
+Landlab.
 
-If you write a Landlab Tutorial or Gist, please contribute it via a pull request.
+Launch notebooks locally
+------------------------
+
+How you installed Landlab determines the steps to launch notebooks that use a
+copy of Landlab on your computer.
+
+User installations
+``````````````````
```

Comment:
@mcflugen are we going to be getting people into trouble if we recommend that they:

1) download the entire source repo to get the environment specification file
2) have them use it to create an environment with binaries
3) then have them use the notebooks within the source repo?

I'm not entirely certain...?

Reply:
The motivation of this PR (first discussed in the March 2 core meeting) is a temporary fix until a more permanent solution for users to get notebooks is implemented. Instructions to get notebooks in this PR follow these [readme instructions](https://github.com/landlab/landlab#conda-environment-with-pre-packaged-binary-distribution), with the exception that this PR instructs to download a release zip rather than cloning, which may be less familiar to some users. If I remember correctly, getting notebooks is not currently in rtd at the moment.